app-misc/screenfetch/screenfetch-3.8.0.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils
```

Comment:
Nothing - so I've removed it.

Note that I didn't add this - the ebuild had it before my change. I just didn't notice it was there unnecessarily.